src/store/store.ts

```diff
@@ -52,34 +57,39 @@ export const loading = (loadingStatus: boolean): LoadingAction => (
   }
 );
 
-export const setState = (todosList: TodoWithUser[], loadingStatus: boolean): SetStateAction => (
-  {
-    type: SET_STATE,
-    todosList,
-    loadingStatus,
-  }
-);
-
-export const alphabeticallyName = (alphabeticallyNameStatus: boolean): AlphabeticallyNameAction => (
-  {
-    type: CHANGE_ALPHABETICALLY_NAME,
-    alphabeticallyNameStatus,
-  }
-);
+export const setState
+  = (todosList: TodoWithUser[], loadingStatus: boolean): SetStateAction => (
+    {
+      type: SET_STATE,
+      todosList,
+      loadingStatus,
+    }
+  );
 
-export const alphabeticallyTitle = (alphabeticallyTitleStatus: boolean): AlphabeticallyTitleAction => (
-  {
-    type: CHANGE_ALPHABETICALLY_TITLE,
-    alphabeticallyTitleStatus,
-  }
-);
-
-export const completedTodo = (completedTodoStatus: boolean): CompletedTodoAction => (
-  {
-    type: CHANGE_COMPLETED,
-    completedTodoStatus,
-  }
-);
+export const sortedName
+  = (sortValue: string, isAlphabeticallyName: boolean): SortedName => (
```

Comment:
please, don't leave `=` on the new line alone, this is not how formatting should look like